tests/kernel/blktests.pm

```diff
@@ -17,6 +17,26 @@ use testapi;
 use utils;
 use repo_tools 'add_qa_head_repo';
 
+#can accept tags: none, all, specific
+sub _test_devices {
+    my ($devices) = @_;
+
+    if ($devices eq 'all') {
+        $devices = script_output('lsblk -d -n -p -o NAME');
```

Comment:
Are you sure you want the tests also to run on the device that contains the root filesystem? 
I think you should avoid this?

Reply:
sure sure :) this crossed my mind but for now I was like... whatever, if SUT dies, well...

Reply:
I started to overengineer all this. I will simply allow 2 options for now, to just have tests running against more devices: listed /dev/* and else with 'nodev'